packages/crypto-wallet-core/src/transactions/xrp/index.ts

```diff
@@ -10,7 +10,7 @@ enum HashPrefix {
 }
 export class XRPTxProvider {
   create(params: {
-    recipients: Array<{ address: string; amount: string }>;
+    recipients: Array<{ address: string; amount: string; recipientsTag?: number }>;
```

Comment:
is there a reason this is `recipientsTag` instead of `recipientTag`? ...or just `tag` so we don't need the `||` below?